server/apiserver/apiserver.go

```diff
@@ -277,7 +285,40 @@ func (c *Client) Read(api *ApiServer) {
 					"message": dpMsg,
 				}).Debug("received ws message")
 
-				// TODO(user team): add user verification here
+				fmt.Println("<Start Validate User>")
+				lastMove, getErr := api.conService.SyncGetLastUserModification(dpMsg.UserID)
```

Comment:
You need to check what the error is here.  If there is no such user, you should register the user.  If there was some other type of dragonboat error that prevented us from completing the operation, you should probably notify the frontend of the error and then return.